templates/cicd/pipeline_cfn.yml

```diff
@@ -166,6 +179,28 @@ Resources:
               - s3:GetBucketLocation
             Resource:
               - "*"
+          {{- if ne .Source.ProviderName "CodeCommit"}}
```

Comment:
nice!

Reply:
I changed it to eq "Bitbucket" for now, until we switch GH over, to appease integ tests.